tests/sentry/api/endpoints/test_organization_user_rollback.py

```diff
@@ -0,0 +1,75 @@
+from django.urls import reverse
+from rest_framework import status
+
+from sentry.models.rollbackorganization import RollbackOrganization
+from sentry.models.rollbackuser import RollbackUser
+from sentry.testutils.cases import APITestCase
+from sentry.testutils.helpers.features import with_feature
+from sentry.testutils.silo import region_silo_test
+
+
+@region_silo_test
+class OrganizationRollbackUserEndpointTest(APITestCase):
```

Comment:
can we add tests for:

- rollback user can't access rollback of an org the user doesn't belong to
- rollback user who isn't currently authed to the organization (i.e. no SSO) does not have access to the rollback
- what else / any other auth edge cases?

Reply:
actually on the second one, i think https://github.com/getsentry/sentry/blob/63a3ef468f463e6de8bc958e14870a6d187f260e/tests/integration/test_api.py#L100 should cover this case.